newIDE/app/src/CommandPalette/CommandManager.js

```diff
@@ -3,18 +3,44 @@ import { type MessageDescriptor } from '../Utils/i18n/MessageDescriptor.flow';
 
 type CommandHandler = () => void | Promise<void>;
 
-export type Command = {|
+export type SimpleCommand = {|
   displayText: MessageDescriptor,
   enabled: boolean,
   handler: CommandHandler,
 |};
 
+export type CommandOption<T> = {|
+  value: T,
```

Comment:
I just realized that this value seems used... nowhere? :D 
Which is understandable, the only thing we need for an option is its `text`, it's `iconSrc` and the function to run, knowing that one `handler` is created for each option, so the closure is already embedding the information of the value.

Unless I missed something, I recommend removing the value here (and so you can remove a bunch of <T> and <OptionType> everywhere 👍) and then everywhere where you were passing a value.

Reply:
Ah yes, I initially put that in as a unique identifier for each option. That was when I was thinking about goto anything, which would have required diffing and comparing options and all sorts of things - and that would have required an explicit unique identifier. But that was eventually scrapped, so yes right now it's unused. I'll remove it, then.

Now that you mention it, the `enabled` prop in Command and its variants is also somewhat unused, since commands that reach the command manager (or scoped command manager) are actually all enabled. `enabled` is really somewhat better suited as a direct hook argument, similar to `commandName`. Should I fix that?

Reply:
> Now that you mention it, the enabled prop in Command (and its related types) is also somewhat unnecessary, since commands that reach the command manager (or scoped command manager) are actually all enabled. enabled is really somewhat better suited as a direct hook argument, similar to commandName. Should I fix that?

Yes I'm all for removing it if it's not actually useful :)

Reply:
Okay, done!